templates/ingress/ingress.yaml

```diff
@@ -1,7 +1,7 @@
 {{- if eq .Values.expose.type "ingress" }}
 {{- $ingress := .Values.expose.ingress -}}
 {{- $tls := .Values.expose.tls -}}
-{{- if eq .Values.expose.ingress.controller "gce" }}
+{{- if or (eq .Values.expose.ingress.controller "gce") (eq .Values.expose.ingress.controller "alb") }}
```

Comment:
I just tested this and it doesnt work (maybe changes since this was first worked on)
It seems alb controller doesnt like wildcard

I was able to get ingress going with changes to the service (svc) templates so that when alb is specified it makes them nodeport, but the ingress rules stay the same